core-features/studio/test.sh

```diff
@@ -21,7 +21,7 @@ yarn prisma studio -p 5555 -b none &
 PRISMA_PID=$!
 
 sleep 3 # Studio takes some time to start up
-yarn jest
+yarn jest --ci --runInBand
```

Comment:
This is going to make it so tests run one at a time (more like Jest will not use workers, but same result).

Not sure how tests were passing earlier, tests with a common DB cannot run in parallel 🤔